### ISSUE TITLE
Infinite reload when the timer runs out instead of a redirect

### ISSUE THREAD

**Issue by aleksandar-todorovic** [proginosko/LeechBlockNG#121]
**Steps to reproduce:**

1. Enter a time period in the settings.
2. Select "within a time period OR after time limit".
3. Add a custom domain (like example.com) to show instead of the blocked sites.
3. Visit a website in the blocklist before the time period starts.
4. After the time period starts, click on any of the internal links on a blocked website (like, revisit the homepage, or click on an article, or whatever).

**Expected result:** I should be redirected to my desired domain.

**Actual result:** The tab just starts infinitely restarting until closed.

Using the beta version of Firefox (70.0b10).

**Comment by LZeph** [comment]
I've had similar problems if the page I'm blocking is Facebook related (Facebook, Instagram etc) and I also have Facebook Container active.

**Comment by proginosko** [comment]
Yes, sounds like an issue with a blocked site embedded within the blocking page. Try removing (or commenting out with # character) sites from the blocklist one-by-one to see if problem is caused by particular sites in the blocklist.
